plugin/pkg/scheduler/factory/factory.go

```diff
@@ -338,6 +338,11 @@ func (f *ConfigFactory) CreateFromConfig(policy schedulerapi.Policy) (*scheduler
 			}
 		}
 	}
+	// Providing HardPodAffinitySymmetricWeight in the policy config is the new and preferred way of providing the value.
+	// Give it higher precedence when it is provided.
```

Comment:
not sure what "than KubeSchedulerConfiguration" means? if it's greater than zero, it'll overwrite the value from cli.